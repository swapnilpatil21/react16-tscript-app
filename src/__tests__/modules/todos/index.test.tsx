import { shallow } from "enzyme";
import Todos from "../../../modules/todos";

describe("<Todos />", ()=> {
  
  it("should render correctly", () => {
    const wrapperComponent = shallow(<Todos />);
    expect(wrapperComponent).toMatchSnapshot();
  })

  it("render child components correctly", () => {
    const wrapperComponent = shallow(<Todos />);

    // console.log(wrapperComponent.debug());
    expect(wrapperComponent.find('AddTodo')).toHaveLength(1);
    expect(wrapperComponent.find('TodoList')).toHaveLength(1);
  })
})

