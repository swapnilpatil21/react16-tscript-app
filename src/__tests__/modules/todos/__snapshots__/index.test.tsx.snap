// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todos /> should render correctly 1`] = `
<Fragment>
  <AddTodo
    addToDoHandler={[Function]}
  />
  <TodoList
    deleteTodoHandler={[Function]}
    todos={Array []}
  />
</Fragment>
`;
